north/src/runtime/linux/mount.rs

```diff
@@ -120,44 +120,36 @@ pub async fn mount(
     let c_target = CString::new(target.as_os_str().as_bytes())?;
     let fstype = fstype.to_string();
     let data = data.map(|s| s.to_string());
-
-    task::spawn_blocking(move || {
-        let c_fstype = CString::new(fstype.as_bytes())?;
-        let c_data = data.as_ref().map_or(ptr::null(), |d| d.as_ptr()) as *const c_void;
-        let result = unsafe {
-            libc::mount(
-                c_source.as_ptr(),
-                c_target.as_ptr(),
-                c_fstype.as_ptr(),
-                flags.bits(),
-                c_data,
-            )
-        };
-
-        match result {
-            0 => Ok(()),
-            _err => Err(anyhow!(
-                "Failed to mount: {}",
-                std::io::Error::last_os_error()
-            )),
-        }
-    })
-    .await
+    let c_fstype = CString::new(fstype.as_bytes())?;
+    let c_data = data.as_ref().map_or(ptr::null(), |d| d.as_ptr()) as *const c_void;
+    let result = unsafe {
+        libc::mount(
+            c_source.as_ptr(),
+            c_target.as_ptr(),
+            c_fstype.as_ptr(),
+            flags.bits(),
+            c_data,
+        )
+    };
```

Comment:
why not spawn on the thread pool without blocking the runtime?

Reply:
There's a issue with calling `libc::mount` from a `spawn_blocking` future. @alfonsoros88 investigates here. The mounting code will be refactored in #106 so calling the mounts in the non blocking context here is temporary fine.